src/parse/asp/builtins.go

```diff
@@ -753,6 +754,64 @@ func addDep(s *scope, args []pyObject) pyObject {
 	return None
 }
 
+func addDatumToTargetAndMaybeQueue(s *scope, target *core.BuildTarget, obj pyObject, systemAllowed, tool bool) {
+	datum := core.NewFileLabel(string(obj.(pyString)), s.pkg)
+	target.AddDatum(datum)
```

Comment:
This isn't quite right. We should be taking in `datum` as a param from `addData()`